### PR TITLE
Fix bugs in UG and DG

### LINKED ISSUE #226: Find latest release sentence in UG needs to be changed
Currently, the UG states `Scroll to the very bottom of the page to find the latest release. Under the Assets section, download the .jar file by clicking on it.`.

The latest release is actually found at the top of the page.

### LINKED ISSUE #227: Change instances of `findingbrudders` to `findingbrUdders`
I attached pictures where the letter U needs to be capitalized. Even the JSON file needs to have a capitalized U.

image

image

In the below picture, the file name also needs to be within a code snippet.

image

### LINKED ISSUE #228: Change the warning for editing JSON file to cover all cases
image

We should also add in that they should not edit the JSON file **after using the application** either, add in any invalid entry, and then open it, which might lead to unexpected behaviour. I think Tester C did just that.

### LINKED ISSUE #229: Inconsistency in examples in UG
image

In the above picture, after the code snippet, the sentence starts with a capital letter, while in the below picture, the sentence starts with an uncapitalized letter.

image

In the following picture for `find` command examples, there has to be a full-stop at the end of the sentence since the other command examples have them too.

image

### PR DESCRIPTION
**Pull request by psh12320**
Fixes #226 , Fixes #227 , Fixes #228 , Fixes #229

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -248,8 +248,8 @@ Finds Udders by specified keywords for each field.
 </box>
 
 **Examples:**
-* `find n/John` returns Udders: `John Mayer` and `John Nissins`<br>
-* `find t/friend` returns `Alex Yeoh` and `Bernice Yu`<br>
+* `find n/John` returns Udders: `John Mayer` and `John Nissins`<br>.
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:251]: Might consider putting fullstop before <br> instead of after

File: docs/UserGuide.md [hunk 2]
```diff
@@ -248,8 +248,8 @@ Finds Udders by specified keywords for each field.
...
+* `find t/friend` returns `Alex Yeoh` and `Bernice Yu`<br>.
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:252]: Same here.

**Review by XinghuaJulia — COMMENTED** [review]
LGTM, minor formatting nits...

**Review by XinghuaJulia — COMMENTED** [review]
LGTM

**Review by ulricolo7 — COMMENTED** [review]
LGTM! Thanks for making the final checks Shri

**Pull request merged**